docs/usage/9.md

```diff
@@ -0,0 +1,20 @@
+Manual Rollback Procedure
+=========================
+If a migration operation was performed without quiescing the application these steps should not be necessary since the application will be running in place on the source cluster. In a case where a quiesced migration operation was performed and rollback is desired for due to failure or some other reason follow the steps below.
+
+Storage
+-------
+If a move operation was performed on an application requiring quiescing care must be taken to prevent data corruption. Ensure that no applications on the destination cluster are running and accessing data on the PVC before trying to bring your applications back up on the source cluster. In a move operation the migration plugin also ensures that the retention policy for bound PVs is to Retain to ensure that data is not lost when an application is cleaned up on the source cluster.
```

Comment:
Suggest

> If a move operation was performed on an application requiring quiescing, care must be taken to prevent data corruption. Ensure that applications on the destination cluster aren't accessing data on the PVC before trying to bring your applications back up on the source cluster.  In a move operation, the migration plugin ensures that the retention policy for bound PVs is 'Retain' to prevent data loss when an application is cleaned up on the source cluster.